cloud_blobstore/__init__.py

```diff
@@ -117,7 +117,9 @@ def upload_file_handle(
             self,
             bucket: str,
             key: str,
-            src_file_handle: typing.BinaryIO):
+            src_file_handle: typing.BinaryIO,
+            content_type: typing.Optional[str] = None,
```

Comment:
I believe the PEP8 way is:

`content_type: typing.Optional[str]=None,`

i.e., no spaces around `=`.

Reply:
Can you also put a `**` argument before `content_type`?